src/session.rs

```diff
@@ -103,7 +103,6 @@ impl SessionManager {
         if !self.sessions.remove(&sess) {
             throw!("session is unmanaged");
         }
-        unsafe { drop(&*sess) };
```

Comment:
Hmm, are you sure no drop of some kind is required here?

Reply:
yeah this is the one I was dubious about, the linter claims this is a no-op. I don't know enough about rust FFI / memory management to comment on this yet. I'll drop the commit for now and keep the lint warning/error